README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@mtyaka Thanks for checking! I think it might make sense to use the tag in the README, since that's what edX uses to mark its releases. That would be `named-release/dogwood.3` though at the moment, I think.

On the other side, we are going to be maintaining our own release branches to support the production instances, so it might make sense to use that here once we have it. @itsjeyd @smarnach Since you're working on related tickets (OC-1554 and OC-1556), what do you think? Maybe @mtyaka just uses the edx/edx-platform tag here, and you update it to our release branch of dogwood in one of your PRs once we have one?